### PR TITLE
Create JsonAdaptedProcedure.java for storage

### PR DESCRIPTION
**Pull request by robinrojh**

### CODE REVIEW

**Comment by teominghuang** [comment]
LGTM, Thank you for putting in so much effort in doing this up!

File: src/main/java/seedu/address/storage/JsonAdaptedProcedure.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+     */
+    public JsonAdaptedProcedure(Procedure source) {
+        information = source.getInfo().info;
+        date = source.getDate().toString();
+        cost = source.getCost().toString();
+    }
+
+    /**
+     * Converts this Jackson-friendly adapted person object into the model's {@code Procedure} object.
+     *
```
Comment (jbkim1999) [inline src/main/java/seedu/address/storage/JsonAdaptedProcedure.java:41]: Should be changed from person to client.

File: src/main/java/seedu/address/storage/JsonAdaptedProcedure.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+     * @throws IllegalValueException if there were any data constraints violated in the adapted person.
```
Comment (jbkim1999) [inline src/main/java/seedu/address/storage/JsonAdaptedProcedure.java:42]: Should be changed from person to client.

**Review by jbkim1999 — APPROVED** [review]
LGTM as well! Just be sure to rename person to client for JavaDocs.

**Pull request merged**